From 72331ff10f1815cb0c1b64dd15a16aa9b14b1559 Mon Sep 17 00:00:00 2001
From: Ken Dreyer <ktdreyer@ktdreyer.com>
Date: Sat, 22 Feb 2014 13:46:01 -0700
Subject: [PATCH 3/3] Fedora: selectively avoid linking libmysqlclient

If libmysqlclient and libtspi are linked to the same binary, we get a
segfault. In XBMC, libavfilter pulls in libgnutls, which pulls in
libtspi.

See https://bugzilla.redhat.com/1069079
---
 configure.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index 2060e24..2a20339 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2898,6 +2898,7 @@ XB_CONFIG_MODULE([xbmc/visualizations/Goom/goom2k4-0],[
   ./configure  \
     CFLAGS="$CFLAGS" \ 
     CXXFLAGS="$CXXFLAGS" \
+    LIBS="$(echo $LIBS | sed -e s/-lmysqlclient//g)" \
     --prefix="${prefix}" --includedir="${includedir}" --libdir="${libdir}" --datadir="${datadir}" \
     --host=$host_alias \
     --build=$build_alias \
@@ -2924,6 +2925,7 @@ XB_CONFIG_MODULE([xbmc/screensavers/rsxs-0.9/], [
     CXX="$CXX" \
     CFLAGS="$TEMPCFLAGS" \ 
     CXXFLAGS="$CXXFLAGS" \
+    LIBS="$(echo $LIBS | sed -e s/-lmysqlclient//g)" \
     `if test "$host_vendor" = "apple"; then echo --with-png=${prefix} --x-includes=/usr/X11/include --x-libraries=/usr/X11/lib; fi` \
     --prefix="${prefix}" --includedir="${includedir}" --libdir="${libdir}" --datadir="${datadir}" \
     --host=$host_alias \
-- 
1.8.5.3

